docs/project-workflow.md

```diff
@@ -1,5 +1,29 @@
 # Project Workflow
 
+## Usage:
+
+`gen3_util projects [OPTIONS] COMMAND [ARGS]...`
+
+  The following `--parameters` and environmental variables are synonymous:
+
+    * OBSERVATION_ID
+    * PATIENT_ID
+    * PROJECT_ID
+    * SPECIMEN_ID
+    * TASK_ID
+
+  You may set them as environmental variables or pass them as parameters to the command line.
+
+    * `--observation_id=observation-1` or `export OBSERVATION_ID=observation-1`
+
+  The following parameters may be used to control system wide behavior:
+
+    * GEN3_UTIL_CONFIG: Path to config file. [default: None]
+    * GEN3_UTIL_FORMAT: Output format. [default yaml]
+    * GEN3_UTIL_STATE_DIR: Path for logs and state information.  [default: ~/.gen3/gen3_util]
+    * GEN3_API_KEY: location of credentials.json file. [default: ~/.gen3/credentials.json]
```

Comment:
Really clear section and great design on being able to use the parameters and environmental variables together.

It might be worth explicitly noting here that the system wide behavior parameters can be also used as environmental variables as well (such as the following) but more than good as is (in fact this might be overly verbose for the README):

```md
  You may also set these as environmental variables or pass them as parameters.

    * `--config=~/.gen3/gen3_config` or `export GEN3_UTIL_CONFIG=~/.gen3/gen3_config`
    * `--format=json` or `export GEN3_UTIL_FORMAT=json`
    * `--state_dir=~/.gen3/gen3_util_custom or `export GEN3_UTIL_STATE_DIR=~/.gen3/gen3_util_custom`
    * `--cred=~/.gen3/custom_credentials` or `export GEN3_API_KEY=~/.gen3/custom_credentials`
```